testing/README.md

```diff
@@ -60,61 +64,109 @@ need to be updated:
-
-1. Add a Cloud Build trigger config YAML file:
-
-  **Pull Request Trigger Config**
-
-  ```YAML
-  name: SAMPLE-pr
-  description: pull-request
-  github:
-    name: cloud-run-samples
-    owner: GoogleCloudPlatform
-    pullRequest:
-      branch: ^master$
-  includedFiles:
-  - SAMPLE_DIR/**
-  filename: SAMPLE_DIR/cloudbuild.yaml
-  ```
-
-  **Nightly Trigger Config**
-
-  Our nightly testing uses "manual" Cloud Build triggers. The Cloud Build Trigger UI is currently the only way to create manual triggers. A current work around is creating a "Push to branch" event trigger. This can be manually triggered and can be updated via the UI to be a "Manually run" trigger.
-
-  ```YAML
-  name: SAMPLE-nightly
-  description: nightly
-  github:
-    name: cloud-run-samples
-    owner: GoogleCloudPlatform
-    push: # TODO: Update when manual triggers are supported
-      branch: nightly
-  includedFiles:
-  - SAMPLE_DIR/**
-  filename: SAMPLE_DIR/cloudbuild.yaml
-  ```
+Each sample with tests requires two Cloud Build triggers:
+
+* A **Pull Request trigger** which checks incoming changes.
+* A **Nightly trigger** which checks the affects of product changes, environment changes, and flakiness.
+
+These triggers are created using trigger configuration files.
+
+1. Create the sample Pull Request trigger config (`pr.trigger-config.yaml`):
+
+   ```yaml
+   name: SAMPLE-pr
+   description: pull-request
+   github:
+     name: cloud-run-samples
+     owner: GoogleCloudPlatform
+     pullRequest:
+       branch: ^master$
+   includedFiles:
+   - SAMPLE_DIR/**
+   filename: SAMPLE_DIR/cloudbuild.yaml
+   ```
+
+1. Create the sample Nightly trigger config (`nightly.trigger-config.yaml`):
+
+   Our nightly testing uses "manual" Cloud Build triggers. The Cloud Build Trigger UI is currently the only way to create manual triggers. A current work around is creating a "Push to branch" event trigger. This can be manually triggered and can be updated via the UI to be a "Manually run" trigger.
+
+   ```yaml
+   name: SAMPLE-nightly
+   description: nightly
+   github:
+     name: cloud-run-samples
+     owner: GoogleCloudPlatform
+     push: # TODO: Update when manual triggers are supported
+       branch: nightly
+   includedFiles:
+   - SAMPLE_DIR/**
+   filename: SAMPLE_DIR/cloudbuild.yaml
+   ```
 
 1. Create a Cloud Build trigger using a config file:
 
-  ```shell
-  gcloud beta builds triggers create github --trigger-config=path/to/trigger-config.yaml
-  ```
+   ```sh
+   gcloud beta builds triggers create github \
+     --trigger-config=sample/path/pr.trigger-config.yaml
+   gcloud beta builds triggers create github \
+     --trigger-config=sample/path/nightly.trigger-config.yaml
+   ```
 
 ## Manually Start Cloud Builds
 
-To manually trigger a Cloud Build from your CLI:
+To manually trigger a Cloud Run (fully managed) build via CLI:
+
+```sh
```

Comment:
We no longer cd into the sample directory for fully managed, as we want to anchor the /workspace at the repo root for access to the templates directory.